inst/NEWS.Rd

```diff
@@ -17,18 +19,27 @@
       \ghit{1017}).
       \item Invisible return object are now supported via new option (Kun Ren
       in \ghpr{1025} fixing \ghit{1024}).
+      \item Unavailable packages referred to in \code{LinkingTo} are now
+      reported (Dirk in \ghpr{1027} fixing \ghit{1026}).
+      \item The \code{sourceCpp} function can now create a debug DLL on
+      Windows (Dirk in \ghpr{1037} fixing \ghit{ghit}).
```

Comment:
Also, there's `\ghit{ghit}`

Reply:
Bah.  Something must be wrong with the water.  Or the coffee.  Or both.

Changed to `\ghit{1035}`. Thanks again.